web/svg/src/jsMain/kotlin/org/jetbrains/compose/web/svg/svg.kt

```diff
@@ -211,10 +211,33 @@ fun ElementScope<SVGElement>.Rect(
     TagElement(
         elementBuilder = Rect,
         applyAttrs = {
-            attr("x", x.toString())
-            attr("y", y.toString())
-            attr("width", width.toString())
-            attr("height", height.toString())
+            x(x)
+            y(y)
+            width(width)
+            height(height)
+            attrs?.invoke(this)
+        },
+        content = content
+    )
+}
+
+@Composable
+@ExperimentalComposeWebSvgApi
+fun ElementScope<SVGElement>.Rect(
+    width: Number,
+    height: Number,
+    rx: Number,
```

Comment:
I convert the bootstrap svg icons to compose dom svg and all the 1800 images don't use ry in their svg data. Adding a required ry parameter would break this converter. So I will add another overload.

Reply:
```
If a properly specified value is provided for rx but not for [ry](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/ry) (or the opposite), then the browser will consider the missing value equal to the defined one.
```
So I am fine with the default parameter